03-command-line-todo-list/lib/cli.rb

```diff
@@ -0,0 +1,73 @@
+require "thor"
+require_relative "database"
+
+module Todo
+  class CLI < Thor
+    def initialize(*args)
+      @store = Todo::Database.new
+
+      super
+    end
+
+    desc "-a or --add [TASK]", "Adds an item into the list of tasks"
+    map %w[-a --add] => :add
+    def add(task)
+      @store.tasks = @store.tasks << task
```

Comment:
Maybe provide `add_task` or something in the store object so that the UI layer is kept as think as possible